data/phoible-references.bib

```diff
@@ -42364,3 +42364,10 @@ @incollection{hercus_yarli_2004
 	Pages = {207--222},
 	Publisher = {John Benjamins},
 	Title = {The Yarli languages}}
+
+@phdthesis{roseslabrada2015mako,
+	Author = {Rosés Labrada, Jorge E.},
+	School = {The University of Western Ontario & Université Lumière-Lyon 2},
+	Title = {The Mako language: Vitality, Grammar and Classification},
+	Url = {https://ir.lib.uwo.ca/cgi/viewcontent.cgi?article=4280&context=etd},
```

Comment:
```suggestion
	Url = {https://ir.lib.uwo.ca/etd/2851},
```

Reply:
(I made this change because it's preferable to link to the metadata page about this dissertation, because it's more likely to be a stable URL than the one for the actual PDF document)